tests/test_ci_app_host_home_isolation.py

```
@@ -476,11 +480,25 @@ def main() -> int:
-        "cmux_reclaim_abandoned_app_host_scopes",
-        "age-bounded process-free scope reclamation",
-    )
+    for unsafe_signal in (
+        '/bin/kill -TERM "$pid"',
+        '/bin/kill -KILL "$pid"',
+    ):
+        if unsafe_signal in APP_HOST_PROCESSES:
+            raise SystemExit(
+                "FAIL: verified app-host cleanup must not signal a reusable PID: "
+                f"{unsafe_signal}"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)